src/hashstore/hashstoreclient.py

```diff
@@ -585,21 +610,31 @@ def get_object_metadata_list(self, origin_directory, num):
         # Create full object list to store into HashStore
         print("Creating list of objects and metadata from metacat db")
         object_metadata_list = []
+        gb_files_to_skip = None
+        if skip_obj_size is not None:
+            gb_files_to_skip = int(skip_obj_size) * (1024**3)
+
         for row in rows:
-            # Get pid, filepath and formatId
-            pid_guid = row[0]
-            metadatapath_docid_rev = origin_directory + "/" + row[1] + "." + str(row[2])
-            metadata_namespace = row[3]
-            row_checksum = row[4]
-            row_checksum_algorithm = row[5]
-            tuple_item = (
-                pid_guid,
-                metadatapath_docid_rev,
-                metadata_namespace,
-                row_checksum,
-                row_checksum_algorithm,
-            )
-            object_metadata_list.append(tuple_item)
+            size = int(row[6])
+            if gb_files_to_skip is not None and size > gb_files_to_skip:
+                continue
+            else:
+                # Get pid, filepath and formatId
+                pid_guid = row[0]
```

Comment:
Thank you for your feedback @jeanetteclark! When I receive the results from the SQL query to retrieve specific columns/values - the column names are not included in the query result set (each row is a tuple item). The only way to access each row would be through the index positions. 

For now, I am going to leave this part as is because it doesn't appear to be a straightforward fix. Should the db schema change, my SQL query will also need to be updated. I will review this section more closely when that happens.

Reply:
sounds good - I wasn't sure what the format would look like on return. Definitely not a critical fix